### DISCUSSION TITLE
So much fun!

### DISCUSSION THREAD

**Discussion by DCElfving** [FundamentalFrequency/.github#43] [Show and tell]
I used walnut for the key plate and brass standoffs rather than stacking layers for the case. I'll eventually add the control labels, but for now, I'm digging the ultra-minimalist (if slightly steam-punky) look. 

Thanks much for sharing your work! Your build guide was easy to follow, and the overall design is clean and thoughtful. 

![lmn1](https://user-images.githubusercontent.com/51248891/177061012-8d39ea1d-2549-4fe0-b59a-bfdc582f185b.jpeg)
![lmn2](https://user-images.githubusercontent.com/51248891/177061019-319d53f0-5af8-49b9-9a4f-d87a7a3eb6ca.jpeg)
![lmn3](https://user-images.githubusercontent.com/51248891/177061020-a650748c-436e-4227-9118-0f5722855615.jpeg)

**Comment by stonepreston** [comment]
This looks so amazing. Great build!
